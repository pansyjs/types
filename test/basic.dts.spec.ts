import { testType } from './utils';
import {
  $Keys,
  $Values,
  $PropertyType,
  $ElementType,
  $Shape,
  $NonMaybeType
} from '../src/basic';

type Props = { name: string; age: number; visible: boolean };

// @dts-jest:group $Keys
{
  // @dts-jest:pass:snap
  testType<$Keys<Props>>();
}