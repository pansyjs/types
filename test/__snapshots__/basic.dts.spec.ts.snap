// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`$Keys testType<$Keys<Props>>() (type) should match snapshot 1`] = `"\\"name\\" | \\"age\\" | \\"visible\\""`;
